### DISCUSSION TITLE
Radioss run script for window version

### DISCUSSION THREAD

**Discussion by kwt0917** [OpenRadioss/OpenRadioss#1014] [Users]
[radioss_run_Script.zip](https://github.com/OpenRadioss/OpenRadioss/files/10453588/radioss_run_Script.zip)

I share the  openradioss run script for window binary version.
You should install the Intel OpenMPI for using mpi version 
(https://www.intel.com/content/www/us/en/developer/articles/tool/oneapi-standalone-components.html#mpi)

You just modify the installed path for OpenRadioss in each *.bat file as below

![image](https://user-images.githubusercontent.com/76669409/213357359-024a8d3a-83fa-4ef9-8109-f2e46d54f800.png)

1. Copy the batch file to input folder
2. double click *.bat file
3. put the input name except _0000.rad
4. put the number of core
5. run

![image](https://user-images.githubusercontent.com/76669409/213359347-c24e878e-49ae-4506-ae8d-5df566f09ffb.png)

Have a good day

**Comment by MarianBulla** [comment]
Hi,
this is really great !
Thank you very much for this script. It helps a lot making the usage easier.
Thanks and best regards,
Marian

**Comment by Timudgin01** [comment]
nothing work :(

**Reply by PaulAltair** [reply]
Hi, what issue are you having specifically?, since this post, we have created some job submission guis, that make it even easier to run under windows: see: [Job Submission guis](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/45252609/python+tk+guis+for+job+submission+anim-vtk+conversion+and+T01-csv+conversion)

**Comment by wmcnCG** [comment]
Had to change a few paths, but this was very helpful - thanks!

**Comment by imdatozdagci** [comment]
How can i visualize .h3d file in paraview?

**Reply by sebastienVilleneuve** [reply]
Hi @imdatozdagci 

It's not possible, 
you need to write animation files ( A001, A002, ....) and convert to vtk using the tool we provide .
https://github.com/OpenRadioss/OpenRadioss/tree/main/tools/anim_to_vtk

Thanks,
Sébastien

**Reply by imdatozdagci** [reply]
![11](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/5d894a1b-c82d-473f-87cf-9f0252008bb4)
Thanks a lot. These are my inputs.  Which files do I need to translate?

**Comment by sebastienVilleneuve** [comment]
@imdatozdagci 

Apparently you didn't generate the A00* files , can you show me what inside 1.rad file ?

Thanks,
Sébastien

**Reply by sebastienVilleneuve** [reply]
you need to use /ANIM instead of /H3D ( an adapt the syntax using the doc) 
https://help.altair.com/hwsolvers/rad/topics/solvers/rad/animation_post_process_h3d_output_r.htm

Sébastien

**Reply by imdatozdagci** [reply]
I'll try and let you know. Thanks a lot
![22](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/2fa973a1-7db6-4090-95bf-8bd952e3fd1b)

**Reply by imdatozdagci** [reply]
Do I need to change anything in the Running Scripts files? Where should i use " /ANIM instead of /H3D"

**Reply by sebastienVilleneuve** [reply]
Just change H3D to ANIM should work in your case ( I think ANIM & H3D have same syntax for keywords you use)

Sébastien

**Comment by imdatozdagci** [comment]
![Ekran görüntüsü 2023-07-19 104946](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/1c139a3d-c6ec-4be6-9bca-2782d3ef2b27)
Here only the line to the file path has H3D and when I change it I get an error. There is no folder named ANIM in the file path.

**Reply by sebastienVilleneuve** [reply]
Hi , you only need to change H3D to ANIM in the 1.rad file

**Reply by imdatozdagci** [reply]
When i run, i take out .out, .rad and Restructured Text Source File. There is any anim file
![55](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/4ce9981f-6445-4252-90d8-ae51f59fdace)

**Reply by sebastienVilleneuve** [reply]
Hi @imdatozdagci 

Yes , you now have the expected animation files. 
You need now, to convert each one to vtk file ( an open in Paraview) .

Sébastien

**Reply by imdatozdagci** [reply]
Thank you very much. You've been very helpful. If I need to bother you one more time, how can I prepare my input file for the animm file when I give a .K file as the input file.

**Reply by sebastienVilleneuve** [reply]
Hi,

You're welcome
Running .k file with the starter will generate a 1.rad file that may content /ANIM options ( depending on options used in .k file)

Sébastien

**Reply by imdatozdagci** [reply]
![66](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/58ad949f-9577-4328-85cf-11c3bb4afb47)
What exactly do you mean starter? I get exactly this screen when I run the .k file normally with the running script.

**Comment by sebastienVilleneuve** [comment]
Radioss solver is in fact 2 executables : starter & engine ( that are run by the script)
in the screenshot you sent ,  not sure the starter ran correctly.

Normally 1.rad is generated automatically by starter executable ( when launching starter with a .k or .key file).

Sébastien

**Comment by imdatozdagci** [comment]
I created a file and put the .k file in it along with the run_mpi file. Even though I run the script from the terminal in the file folder (test.k without deleting anything), it gives an error without creating a .rad file. What do you think could be the reason?

**Reply by sebastienVilleneuve** [reply]
Hi @imdatozdagci 

can you send the .k fle ? I will have a look

Thanks,
Sébastien

**Comment by imdatozdagci** [comment]
[Impact_AP_meshed_0000.zip](https://github.com/OpenRadioss/OpenRadioss/files/12095831/Impact_AP_meshed_0000.zip)
We try 2 different .k file.  You are very kind person, thank you very much

**Reply by sebastienVilleneuve** [reply]
[1.rad_files.tar.gz](https://github.com/OpenRadioss/OpenRadioss/files/12095894/1.rad_files.tar.gz)

here the 1.rad files I get running the starter

Maybe something taht I don't understand with the script ( I'm lauching directly starter/engine by hand)

**Comment by imdatozdagci** [comment]
Thank you very much. When I try to do it myself I get an error like this. Do you have any solution suggestions?
![10](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/cf2c5991-7c8e-4cae-8f8b-9b585b1c4c5e)

**Reply by izy-glitch** [reply]
Yes I also need the answer to this question.

**Reply by PaulAltair** [reply]
This script is hardcoded to expect _0000.rad input... It could be modified, but there are already some much easier to use guis now that will allow you to run .k or .key input files already: see: [Gui submission tool for Windows/Linux](https://github.com/orgs/OpenRadioss/discussions/1231#discussioncomment-6490163)

**Reply by imdatozdagci** [reply]
Thank you very much, I got everything done thanks to you.

**Reply by Musibra** [reply]
Hello sir, 
 I tried this GUI, its very good and easy but, when give K file as input, only generates anim files. How can i change output type to H3D format, because I want to view in HyperView

**Reply by PaulAltair** [reply]
The DYNA (k/key) input always creates ANIM by default, you can read those in HyperView too... but if you want h3d instead, you can do following:

When you run the job, you will see, that for your modelname.k the starter automatically creates a modelname_0001.rad (and if a 'modelname_0001.rad' exists, it will be overwritten)

You can modify that modelname_0001.rad to add the h3d requests you want, then submit that to gui (or from command line)

So:

1: Run the modelname.k
2: as soon as engine is running, stop/kill it
3: add h3d requests to the modelname_0001.rad (or add your own predefined modelname_0001.rad)
4: submit the modelname_0001.rad file in the gui or from commandline
5: run will then generate the h3d.

remember, starting again with k/key will overwrite the 0001.rad file, so maybe a good idea to save a copy of it as e.g. modelname_0001.txt

**Reply by Musibra** [reply]
thank you

**Comment by janakrajawasthi** [comment]
Hello Everyone,
I want to use OpenRadioss for my Research at University. I have tried many times to download and install Openradioss. Unfortunately, I'm unable to install OpenRadioss on my Windows 11 laptop. 
Can anyone of you send me step-by-step instructions for downloading and installing OpenRadioss for Windows 11 64x? Your Help is highly appreciated.

Regards,
Janak

**Reply by MarianBulla** [reply]
Hello Janak,
thank you very much in your interest in OpenRadioss.

I have created a bunch of YouTube videos on how to install OpenRadioss on Windows or Linux: https://www.youtube.com/@OpenRadiossCommunity

Are you trying to compile the source code or using the compiled executable ?

There is also a detailed How-To on our GitHub page: https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md 

Hope it helps you ?

Thanks and best regards,
Marian
